### PR TITLE
Add Dribbble provider

### PR DESCRIPTION
**Pull request by victorsalesdev**
This implements Dribbble's OAuth provider.

### CODE REVIEW

File: src/providers/dribbble.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+		}
+	): Promise<URL> {
+		return await this.client.createAuthorizationURL({
+			state,
+			scopes: options?.scopes ?? []
+		});
+	}
+
+	public async validateAuthorizationCode(code: string): Promise<DribbbleTokens> {
+		const result = await this.client.validateAuthorizationCode<AuthorizationCodeResponseBody>(
```
Comment (pilcrowonpaper) [inline src/providers/dribbble.ts]: You shouldn't need to pass `AuthorizationCodeResponseBody` since that's already the default type value

**Review by pilcrowonpaper — APPROVED** [review]
Can you create a new minor changeset? (See CONTRIBUTING.md)

**Comment by victorsalesdev** [comment]
Sorry, I just realized that I was supposed to open an issue first. Let me know if you need that.

**Review by pilcrowonpaper — APPROVED** [review]

**Comment by pilcrowonpaper** [comment]
Thanks!

**Pull request merged**